src/lighting/lighting-controller.js

```diff
@@ -368,6 +372,9 @@ class LightingController {
         _self.request.time = moment().add(5, 'minutes');
         _self.turnOffLights();
         break;
+      case 'color':
+          _self.colorLights(_self.request.color);
```

Comment:
__self.request.color_ doesn't need to be passed into the _colorLights_ method, because it is part of the global __self.request_ object. Please remove the parameter (to remain consistent with the other methods), and just reference __self.request.color_ within the _colorLights_ method (instead of _xColor_).